README.md

```diff
@@ -42,6 +42,9 @@ pip install git+git://github.com/ionpath/master
 ```
 
 ## Usage
+To use the MIBItracker API, you will need to use the backend url listed in the
```

Comment:
Nitpicky, but this is a bit of a run on sentence. Maybe "...in the About page. This can be accessed..."